pkg/resources/service/service_test.go

```diff
@@ -15,49 +15,97 @@
 package service
 
 import (
+	"fmt"
+	"net"
 	"os"
+	"strings"
 	"testing"
+	"time"
 
 	"github.com/stretchr/testify/assert"
 	"github.com/triggermesh/tm/pkg/client"
 )
 
+const (
+	// timeout in seconds to check that resulting service is reachable
+	dialTimeout = 10
```

Comment:
Same comment as before about making this a `Duration` or at least adding the unit to the const name.